lib/src/models/message_options.dart

```diff
@@ -11,6 +11,7 @@ class MessageOptions {
     this.onPressAvatar,
     this.onLongPressAvatar,
     this.onLongPressMessage,
+    this.onTapDownMessage,
```

Comment:
Would it not be better to create a subclass of message_options in our code-base with the added fields?